src/test/scala/com/amazon/deequ/profiles/ColumnProfilerTest.scala

```diff
@@ -400,4 +400,57 @@ class ColumnProfilerTest extends WordSpec with Matchers with SparkContextSpec
     }
   }
 
+  "return correct profile for the Titanic dataset" in withSparkSession { session =>
+    val data = session.read.format("csv")
+                .option("inferSchema", "true")
+                .option("header", "true")
+                .load("test-data/titanic.csv")
+    
+    val columnProfiles = ColumnProfiler.profile(data)
+
+    val idProfile = columnProfiles.profiles("PassengerId")
+    assert(idProfile.column == "PassengerId")
+    assert(idProfile.dataType == DataTypeInstances.Integral)
+    assert(idProfile.completeness == 1.0)
+    assert(idProfile.approximateNumDistinctValues == 887) //exact is 891
```

Comment:
This value might vary from run to run, better to check whether its in a given interval